clients/include/utility.hpp

```diff
@@ -2219,6 +2219,408 @@ inline void host_bsrmv(hipsparseDirection_t dir,
     }
 }
 
+template <typename T>
```

Comment:
it's a copy of host_gebsrmv in rocsparse right?

Reply:
(from rocsparse)

Reply:
Yup